apps/src/templates/projects/PersonalProjectsTable.jsx

```diff
@@ -0,0 +1,298 @@
+          format: this.thumbnailFormatter,
+          props: {style: {
+            ...styles.cell,
+            ...styles.cellThumbnail
+          }}
+        }
+      },
+      {
+        property: 'name',
+        header: {
+          label: i18n.projectName(),
+          props: {style: {
+            ...styles.headerCell,
+            ...styles.headerCellName
+          }},
+        },
+        cell: {
+          format: this.nameFormatter,
+          props: {style: {
+            ...styles.cell,
+            ...styles.cellName
+          }}
+        }
+      },
+      {
+        property: 'type',
+        header: {
+          label: i18n.projectType(),
+          props: {style: styles.headerCell},
+          transforms: [sortable],
+        },
+        cell: {
+          format: this.typeFormatter,
+          props: {style: {
+            ...styles.cellType,
+            ...styles.cell
+          }}
+        }
+      },
+      {
+        property: 'updatedAt',
+        header: {
+          label: i18n.lastEdited(),
+          props: {style: styles.headerCell},
+          transforms: [sortable],
+        },
+        cell: {
+          format: this.dateFormatter,
+          props: {style: styles.cell}
+        }
+      },
+      {
+        property: 'isPublished',
+        header: {
+          label: i18n.publicGallery(),
+          props: {style: styles.headerCell},
+        },
+        cell: {
+          format: this.isPublishedFormatter,
+          props: {style: {
+            ...styles.cell,
+            ...styles.cellIsPublished
+          }}
+        }
+      },
+      {
+        property: 'actions',
+        header: {
+          label: i18n.quickActions(),
+          props: {style: styles.headerCell},
+        },
+        cell: {
+          format: this.actionsFormatter,
+          props: {style: {
+            ...styles.cell,
+            ...styles.cellAction
+          }}
+        }
+      }
+    ];
+  },
+
+  render() {
+    const sortableOptions = {
+      // Dim inactive sorting icons in the column headers
+      default: {color: 'rgba(0, 0, 0, 0.2 )'}
+    };
+
+    // Define a sorting transform that can be applied to each column
+    const sortable = wrappedSortable(this.getSortingColumns, this.onSort, sortableOptions);
+    const columns = this.getColumns(sortable);
+    const sortingColumns = this.getSortingColumns();
+
+    const sortedRows = sort.sorter({
+      columns,
+      sortingColumns,
+      sort: orderBy,
+    })(convertChannelsToProjectData(this.props.projectList));
+
+    return (
+      <Table.Provider
+        columns={columns}
+        style={styles.table}
+      >
+        <Table.Header />
+        <Table.Body rows={sortedRows} rowKey="channel" />
+      </Table.Provider>
+    );
+  }
+});
+
+// The project table uses the channels API to populate the personal projects
+// and the data needs to be filtered and mapped before displaying.
+const convertChannelsToProjectData = function (projects) {
```

Comment:
I think I would avoid having the component be responsible for converting the data. Instead I would have the caller (it looks like you don't have any callers yet) do the conversion, and pass this component data in the converted form. This would presumably also affect your story.